recipe/build.sh

```diff
@@ -11,7 +11,8 @@ if [[ "$target_platform" == osx-arm64 ]]; then
 		-DCMAKE_INSTALL_LIBDIR=lib \
 		-DCMAKE_BUILD_TYPE=Release \
 		-DCMAKE_VERBOSE_MAKEFILE=ON \
-		-DENABLE_TESTING=Off
+		-DENABLE_TESTING=OFF \
+  		-DNNG_TOOLS=ON
```

Comment:
```suggestion
		-DNNG_TOOLS=ON
```

this is currently mix between spaces and tabs
<img width="335" alt="image" src="https://github.com/conda-forge/nng-feedstock/assets/29506042/e0b8bc6d-5aff-4647-8bf7-4df9849af824">